emperor/support_files/vendor/js/three.js-plugins/SVGRenderer.js

```diff
@@ -339,20 +372,21 @@ THREE.SVGRenderer = function () {
 		var scaleX = element.scale.x * _svgWidthHalf;
 		var scaleY = element.scale.y * _svgHeightHalf;
 
-		_svgNode = getRectNode( _rectCount ++ );
+		if ( material.isPointsMaterial ) {
+			scaleX *= material.size;
```

Comment:
why sometimes you add 2 new lines and some others not? BTW I'm not a fan of the double lines ...

Reply:
Most of the changes to SVGRenderer.js come from the THREE.js repo, I didn't write most of this code. There's one minor exception, I found a bug in the way it deals with transparent objects, and I added that in https://github.com/biocore/emperor/pull/602/commits/27c02250dfd63846c518e1ae88d92da8c89cf36c and I also submitted a PR to their repo (which already got merged):

https://github.com/mrdoob/three.js/pull/12071

Reply:
got it, thanks.